### PR TITLE
Rerun analysis with Zernike features

### PR DESCRIPTION
**Pull request by jenna-tomkinson**
This PR makes minor changes where I add calculating Zernikes for both magnitudes and phases (in the `MeasureObjectIntenistyDistribution` module) for both pipelines.

I reran the pipelines to output new SQLite files and reran the processing of the features to get new parquet files (e.g., annotated, normalized, and feature selected).

### CODE REVIEW

File: 2.cellprofiler_analysis/NF1_analysis_3channel.cppipe [hunk 1]
```diff
@@ -207,12 +207,12 @@ MeasureObjectNeighbors:[module_num:13|svn_version:'Unknown'|variable_revision_nu
     Name the output image:PercentTouching
     Select colormap:Oranges
 
-MeasureObjectIntensityDistribution:[module_num:14|svn_version:'Unknown'|variable_revision_number:6|show_window:False|notes:['Measure object intensity distributions witin objects using center x,y coords for each single object.', '', 'Currently we do not measure the Zernikes.']|batch_state:array([], dtype=uint8)|enabled:True|wants_pause:False]
+MeasureObjectIntensityDistribution:[module_num:14|svn_version:'Unknown'|variable_revision_number:6|show_window:False|notes:['Measure object intensity distributions witin objects using center x,y coords for each single object.', '', "We calculate Zernikes for magnitudes and phases as we are interested in seeing phase even if it doesn't help with classifying phenotypes."]|batch_state:array([], dtype=uint8)|enabled:True|wants_pause:False]
```
Comment (MikeLippincott) [inline 2.cellprofiler_analysis/NF1_analysis_3channel.cppipe:210]: good `.cpipe` documentation

**Review by MikeLippincott — APPROVED** [review]
Only comment I have is wondering if `.parquet` files are supposed to be `.gitignore`'d? Other than that LGTM! Ready to merge!

File: 2.cellprofiler_analysis/NF1_analysis_3channel.cppipe (hunk 1 above)
Comment (jenna-tomkinson) [reply 2.cellprofiler_analysis/NF1_analysis_3channel.cppipe:210]: Ty!

**Comment by jenna-tomkinson** [comment]
Great point! I am including the `parquet` files so that they are easily accessible for analysis, and they are pretty small to include.

@MikeLippincott Thank you for the review, I am merging now!

**Pull request merged**